### PR TITLE
[FIX] website: make website search box not editable

### PR DESCRIPTION
**Pull request by alsh-odoo**
This traceback occurs when the user tries to edit the website searchbar

To reproduce this issue:-

1) Install `website_sales`
2) open `website` and click on the `search icon`
3) A `website search box` will popup
4) Now open the `editor` and try to change the `inline text` 
5) Save the record
6) A traceback appears

Error:- 
```
ValueError: not enough values to unpack (expected 1, got 0)
```

The issue is occurring because on `web search box`  we are getting 
section_xpath - `/data/xpath[3]/form/t[1]/div[1]/button[1]/i[1]` and 
we are not getting any value in root from this `section_xpath` 

over here -

https://github.com/odoo/odoo/blob/cdedc1752b845bed9acc0dd4bfeb4b343b675326/addons/web_editor/models/ir_ui_view.py#L121

To fix this issue `o_not_editable` class has been used on that website search bar

sentry-3933794130

### CODE REVIEW

**Comment by alsh-odoo** [comment]
Hello @odoo/sentry_reviewers

Can I have a review on this PR.

Thank You :slightly_smiling_face:

**Comment by alsh-odoo** [comment]
Hello @rdeodoo 

I just given the `inline text` as reference to produce this issue.

If we try to edit anything for the `website search box` the path is broken and it leads to the exception.

**Comment by rdeodoo** [comment]
But if you hide it, you won't be able to edit anything, right?

File: addons/website/static/src/snippets/s_searchbar/000.js [hunk 1]
```diff
@@ -251,6 +251,11 @@ publicWidget.registry.searchBar = publicWidget.Widget.extend({
      * @private
      */
     _onFocusOut: function () {
+        // If focus is away from the search modal, hide it.
+        const modal = Modal.getOrCreateInstance(this.$input.closest('#o_search_modal'));
+        if (modal) {
+            modal.hide()
+        }
```
Comment (rdeodoo) [inline addons/website/static/src/snippets/s_searchbar/000.js:258]: Wasn't the `css_editable_mode_hidden` xml solution I [explained](https://github.com/odoo/odoo/pull/164129#pullrequestreview-2035205461) working?

Comment (alsh-odoo) [reply addons/website/static/src/snippets/s_searchbar/000.js:258]: No it is not working

Comment (rdeodoo) [reply addons/website/static/src/snippets/s_searchbar/000.js:258]: What is not working? I had a quick look on runbot, and by adding `css_editable_mode_hidden` on the `#o_search_modal` in the DOM, then entering edit mode, it is hidden (as if it was not opened in the first place)?
What issue did you face

**Review by rdeodoo — APPROVED** [review]
@robodoo r+

Not fully stable friendly as without module update existing DB won't have the fix but it's already something.

**Pull request closed without merging**